css/lodesiempre.html

```diff
@@ -19,24 +17,168 @@
 
   <div class="header-background">
     <div class="column-container">
-        <h1 class="primary-font white-color margin-title">Apoya a los comercios gastronomicos.</h1>
-        <h3 class="secondary-font white-color margin-subtitle">Compra hoy un voucher con descuento, y ayuda a tu comercio favorito a surfear la ola.</h3>
+        <h1 class="primary-font white-color margin-title">Apoyá a los comercios gastronómicos.</h1>
+        <h4 class="secondary-font white-color margin-subtitle">Compra hoy un voucher con descuento, y ayudá a tu comercio favorito a surfear la ola.</h4>
         <h1 class="primary-font primary-color margin-title">417 personas</h1>
-        <h3 class="secondary-font white-color margin-subtitle"> ya apoyaron esta causa</h3>
-        <h2 class="primary-font primary-color margin-subtitle">80 restaurantes adheridos</h2>
+        <h4 class="secondary-font white-color margin-subtitle"> ya apoyaron esta causa</h4>
+        <h3 class="primary-font primary-color margin-subtitle">80 restaurantes adheridos</h3>
+    </div>
+  </div>
+
+  <div class="row-container">
+    <img src="onboarding-assets/img/donacion.png" class="third-width column-container">
+    <div class="column-container left-align-flex third-width">
+      <h2 class="primary-font primary-color margin-subtitle">Por cada voucher vendido</h2>
+      <h1 class="primary-font secondary-color margin-subtitle">FNC donara 1 vianda</h1>
+      <h3 class="primary-font primary-color margin-subtitle">a instituciones seleccionadas.</h3>
+      <h5 class="secondary-font body-color margin-button"> Nos importa la salud y el impacto socio economico en la comunidad. El proposito de este proyecto es darnos aliento, que codo con codo salimos adelante.</h5>
```

Comment:
this is more a paragraph `<p>` than a title element. Check below all that are similar

Reply:
if I define all elements as a `<p>` how do I set the font-size accordingly without using a bunch of extra classes?
Isn't it better if I use h1..5 with the font-sizes that I previously defined?